### PR TITLE
sig-windows contributing.md updates

### LINKED ISSUE #6429: Update sig-windows CONTRIBUTING.md
### Describe the issue

https://github.com/kubernetes/community/blob/master/sig-windows/CONTRIBUTING.md should be reviewed and updated accordingly. 

/sig windows
/assign @marosset @jayunit100 @jsturtevant @claudiubelu

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

**Which issue(s) this PR fixes**:

Fixes #6429

### CODE REVIEW

File: sig-windows/CONTRIBUTING.md [hunk 1]
```diff
@@ -52,30 +53,36 @@ If you would like to build all binaries at once, then run `./build/run.sh make c
...
 
-### Updating the Node binaries
+### Testing your changes
 
-Once you have binaries built, the easiest way to test them is to replace them on an existing cluster. This section assumes you already have a cluster in the cloud of your choice. To update the binaries on an existing node, follow these steps:
+#### Build a local cluster
 
-1. Drain & cordon a node with `kubectl drain <nodename>`
+The easist way to test changes is to use the community maintained [SIG Windows developer environment](https://github.com/kubernetes-sigs/sig-windows-dev-tools#welcome-to-the-sig-windows-development-environment) project.
+You can use this to build a fully functione cluster locally with binaries built from source.
```
Comment (jsturtevant) [inline sig-windows/CONTRIBUTING.md:61]:
```suggestion
You can use this to build a fully functional cluster locally with binaries built from source.
```

**Review by jsturtevant — COMMENTED** [review]
lgtm, one minor spelling

File: sig-windows/CONTRIBUTING.md [hunk 2]
```diff
@@ -26,21 +26,22 @@ View the leadership team in SIG-Windows and other subprojects in the [getting st
...
 
 ## Building Kubernetes for Windows from Source
 
-The Kubernetes build scripts have not been ported to Windows, so it's best to run in a Linux VM where you can run the same Docker container used in the official Kubernetes builds. This simplifies the steps, but means that you cannot build under Windows Subsystem for Linux (WSL). 
+The Kubernetes build scripts have not been ported to Windows, so it's best to develop in a Linux VM or [WSL2](https://docs.microsoft.com/windows/wsl/) environment where you can run the same Docker container used in the official Kubernetes builds.
 
-It's best to skim over the [Building Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/build/README.md) guide if you have never built Kubernetes before to get the latest info. These steps are a summary focused on cross-building the Windows node binaries (kubelet & kube-proxy).
+It's best to skim over the [Building Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/build/README.md) guide if you have never built Kubernetes before to get the latest info. These steps are a summary focused on cross-building the Windows node binaries (kubectl, kubelet, and kube-proxy).
```
Comment (aravindhp) [inline sig-windows/CONTRIBUTING.md:38]: I suggest saying `read through` rather `skim over`. Skim over sometimes becomes skip over for the reader 😆

File: sig-windows/CONTRIBUTING.md [hunk 3]
```diff
@@ -52,30 +53,36 @@ If you would like to build all binaries at once, then run `./build/run.sh make c
...
+
+#### Updating the Node binaries
+
+If you already have an existing cluster you also test your changes by swapping the binaries on a node.
+To update the binaries on an existing node, follow these steps:
+
+1. Drain & cordon a node with `kubectl drain <nodename>`.
 2. Connect to the node with SSH or Windows Remote Desktop, and start PowerShell
-3. On the node, run `Stop-Service kubelet -Force`
+3. On the node, run `Stop-Service kubelet -Force`.
```
Comment (aravindhp) [inline sig-windows/CONTRIBUTING.md:70]: Shouldn't this be `Stop-Service kubelet -Force` and ``Stop-Service kube-proxy -Force`

File: sig-windows/CONTRIBUTING.md [hunk 4]
```diff
@@ -52,30 +53,36 @@ If you would like to build all binaries at once, then run `./build/run.sh make c
...
 4. Copy kubelet.exe and kube-proxy.exe to a cloud storage account, or use SSH to copy them directly to the node.
```
Comment (aravindhp) [inline sig-windows/CONTRIBUTING.md:71]: I suggest saying `scp the binaries directly to the node`. Mentioning a cloud storage account might muddy the waters.

**Review by aravindhp — COMMENTED** [review]
Thanks for updating this, @marosset. Mostly lgtm, just a few suggestions.

File: sig-windows/CONTRIBUTING.md (hunk 3 above)
Comment (marosset) [reply sig-windows/CONTRIBUTING.md:70]: Yes.
I actually want to delete the troubleshooting info from this doc once we have all the refactors on the website completed.
I can make this simple fix tho.

File: sig-windows/CONTRIBUTING.md (hunk 4 above)
Comment (marosset) [reply sig-windows/CONTRIBUTING.md]: updated

**Comment by marosset** [comment]
Updated based on feedback from @aravindhp - Thanks!

**Pull request merged**